GNUmakefile

```diff
@@ -7,7 +7,9 @@ GOTOOLS = \
 	golang.org/x/tools/cmd/cover \
 	golang.org/x/tools/cmd/stringer \
 	github.com/axw/gocov/gocov \
-	gopkg.in/matm/v1/gocov-html
+	gopkg.in/matm/v1/gocov-html \
+	github.com/golang/protobuf/protoc-gen-go \
```

Comment:
We already vendor `gogo/protobuf` (although probably not the compiler) and use that via a few other dependencies. Is there value in sticking with `gogo` fork everywhere?

Reply:
Cool! Sounds good and done.